### PR TITLE
Allow jpg and png files to be previewied and printed

### PR DESCRIPTION
**Pull request by PhongDiep2003**

### CODE REVIEW

File: api/config/config.example.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "passwordStrength": "medium",
   "PRINTING": {
-    "ENABLED": false
+    "ENABLED": true
```
Comment (evanugarte) [inline api/config/config.example.json]: undo this

File: src/Enums.js [hunk 2]
```diff
@@ -50,6 +50,8 @@ function membershipStateToString(accessLevel) {
 
 const BASE_API_URL = process.env.REACT_APP_BASE_API_URL || 'http://localhost:8080/';
 
+
```
Comment (evanugarte) [inline src/Enums.js]: undo the changes in this file

File: src/APIFunctions/2DPrinting.js [hunk 3]
```diff
@@ -81,7 +81,7 @@ export async function printPage(data, token) {
   await axios.post(url.href, data, {
     headers: {
       'Content-Type': 'multipart/form-data',
-      'Authorization': `Bearer ${token}`
+      'Authorization': `Bearer ${data.token}`
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js]: if we are using data.token, remove token as a param to the `printPage` function

File: src/Pages/2DPrinting/2DPrinting.js [hunk 4]
```diff
@@ -97,9 +97,58 @@ export default function Printing(props) {
...
-      getUri();
+      // get the file type
+      const mediaType = dataUrl.split(';')[0].split(':')[1].split('/')[1];
+      // if the file type is pdf
+      if (
+        mediaType === 'pdf'
+      ) {
+        getUri();
+      } else if ( // if the file type is an image
+        mediaType === 'jpg' || mediaType === 'png' || mediaType === 'jpeg'
```
Comment (evanugarte) [inline src/Pages/2DPrinting/2DPrinting.js]:
```suggestion
        ['jpg',  'png',  'jpeg'].includes(mediaType)
```

**Review by evanugarte — APPROVED** [review]

**Review by akshtshrma24 — APPROVED** [review]
